libs/model/src/models/contract_abi.ts

```diff
@@ -26,8 +26,8 @@ export default (
     'ContractAbi',
     {
       id: { type: dataTypes.INTEGER, primaryKey: true, autoIncrement: true },
-      nickname: { type: dataTypes.STRING, allowNull: true },
-      abi: { type: dataTypes.JSONB, allowNull: false },
+      nickname: { type: dataTypes.STRING, allowNull: true, unique: true },
+      abi: { type: dataTypes.JSONB, allowNull: true },
```

Comment:
This is one of those cases where switching from `false` to `true` is dangerous. While the database indeed does not enforce `NOT NULL` we should migrate to do so rather than updating the model. The entire point of this table is to store ABIs so if ABI is null that breaks a lot of different functionality.

Reply:
added to migration